### PR TITLE
enable mongodb metrics collection and whitelisting

### PR DESCRIPTION
**Pull request by dragon9786**
cc: @leoxlin and @mdanzinger for context. Do not merge yet until we have verified that the metrics from agent are scrapable.

### CODE REVIEW

File: cmd/do-agent/config.go [hunk 1]
```diff
@@ -116,6 +117,9 @@ func init() {
 	kingpin.Flag("dbaas-metrics-path", "enable DO DBAAS metrics collection (this must be a DO DBAAS metrics endpoint)").
 		StringVar(&config.dbaas)
 
+	kingpin.Flag("mongodb-metrics-path", "enable mongodb metrics collection").
```
Comment (leoxlin) [inline cmd/do-agent/config.go:120]: Let's do `enable DO DBAAS mongodb metrics collection (this must be a DO DBAAS metrics endpoint)` like the previous argument to make it clear it is a flag for DO metrics.

Comment (dragon9786) [reply cmd/do-agent/config.go]: done

**Comment by leoxlin** [comment]
I missed this in the first review but the new config is not actually being utilized. You will need to also hook up the aggregator / scraper See:

https://github.com/digitalocean/do-agent/blob/46d9ffa83bd42201537de94cbdcebcc71d1a9fdc/cmd/do-agent/config.go#LL286C5-L286C17

https://github.com/digitalocean/do-agent/blob/46d9ffa83bd42201537de94cbdcebcc71d1a9fdc/cmd/do-agent/config.go#L227

**Review by leoxlin — APPROVED** [review]
LGTM w/ @prichrd

**Pull request merged**